src/utils.jl

```diff
@@ -66,11 +66,23 @@ Convert any geometry or collection of geometries into a flat
 vector of `Tuple{Tuple{Float64,Float64},Tuple{Float64,Float64}}` edges.
 """
 function to_edges(x, ::Type{T} = Float64) where T
+    
     edges = Vector{TupleEdge{T}}(undef, _nedge(x))
     _to_edges!(edges, x, 1)
     return edges
 end
 
+function find_point_constructor(x, ::Type{T})
+    if _ismeasured(x)
+        SVPoint_4D
+    elseif _is3d
+        SVPoint_3D
+    else
+        SVPoint_2D
+    end
+
+end
```

Comment:
```suggestion
function find_point_constructor(x, ::Type{T})
    M = _ismeasured(x)
    Z = _is3d(x)
    N = 2 + M + Z
    return SVPoint{N, T, Z, M}
end
```
This is what we should have, IMO, but it does overlap in purpose with the SVPoint_4D etc constructors defined earlier.  I wonder whether we should simply define a default and a fallback for three-dimensional points, since that's where all the uncertainty is...

Reply:
What do you mean? I don't understand.